data/htop/osfamily/RedHat.yaml

```diff
@@ -0,0 +1,5 @@
+---
+  nagios-plugins::settings:
```

Comment:
This file is wrong, here and probably shouldn't exist

Reply:
removed